pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -1629,7 +1629,7 @@ func registerCDC(r registry.Registry) {
 	r.Add(registry.TestSpec{
 		Name:             "cdc/kafka-azure",
 		Owner:            `cdc`,
-		CompatibleClouds: registry.AllExceptAWS,
+		CompatibleClouds: registry.OnlyAzure,
 		Cluster:          r.MakeClusterSpec(2, spec.GCEZones("us-east1-b")),
```

Comment:
nit: `spec.GCEZones("us-east1-b")` isn't doing anything if we're running on only Azure. You'd need to add spec.AzureZones but I think you might not need it at all from skimming the test.

Reply:
Yep, `spec.AzureZones` doesn't exist as of yet, so it will default to `defaultLocations[0]`,

```
// These default locations support availability zones. At the time of
// this comment, `westus` did not.
var defaultLocations = []string{
	"eastus",
	"westus2",
	"westeurope",
}
```

We can add `spec.AzureZones` at a later time since the default isn't likely to change any time soon.